.github/workflows/ic-design-system-branches.yml

```diff
@@ -6,6 +6,10 @@ on:
       - main
       - gh-pages
 
+concurrency:
+  group: ci-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Would we not want this to be false as we don't want to cancel in progress builds?